manifests/examples/virtualbox/stretch-vagrant.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: debian-{system.release}-{system.architecture}-{%y}{%m}{%d}
+provider:
+  name: virtualbox
+  guest_additions: /usr/share/virtualbox/VBoxGuestAdditions.iso
+bootstrapper:
+  workspace: /target/stretch
```

Comment:
I added unique folder names to make it easier to run vagrant after multiple builds (jessie, stretch, sid). After each build, I'll run

```Shell
vagrant box add <build_name> <box_file>
vagrant init <build_name>
vagrant up
```

This can't be done when multiple builds are being sent to the same folder.